cmd/cmd.go

```diff
@@ -34,7 +34,6 @@ import (
 	"github.com/ollama/ollama/api"
 	"github.com/ollama/ollama/auth"
 	"github.com/ollama/ollama/format"
-	"github.com/ollama/ollama/parser"
```

Comment:
Why did we jam this in a `types/` module? It's confusing to me as this mixes up our model data with the parsing. Any contributor looking for the parser might be lost now - we should keep things flat and simple here

Reply:
My understanding is `types/` should primarily be for storing common data structures we use